LiskHQ/lisky
# Fix lockfile usage - Closes #529

### What was the problem?
When in non interactive mode, it was unable to pipe commands because of the lockfile.

### How did I fix it?
Only use lockfile when it needs to.
In this case, only when writing to config file, it is locked.

### How to test it?
```
./bin/lisky create transaction 0 200 123L --no-signature --json | ./bin/lisky sign transaction
```
and check if there is no error.
### Review checklist

* The PR solves #529 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
